### ISSUE TITLE
Returning KL divergence

### ISSUE THREAD

**Issue by Wilco17** [CannyLab/tsne-cuda#75]
Thank you for this fantastic work!

Could it be possible the _fit_transform()_ method returns the KL divergence of the run?

Thx!

**Comment by stu-blair** [comment]
related question, @DavidMChan , is the `Avg. Gradient Norm` printed to the log the same/analogous to KLD ?

**Comment by DavidMChan** [comment]
The average gradient norm is the norm of the gradient of the KL-divergence with respect to the particle positions. Thus, it can be a proxy for how stable the optimization process is, but is not the same as the KL.

**Comment by stu-blair** [comment]
Thanks for the explanation! That makes sense.

So, just to confirm-- there's no way at all currently to see the KL/D value ?

**Comment by DavidMChan** [comment]
Currently, no - but I'll consider working it into the next version, and we're always welcome to PRs if anyone wants to contribute!
